scripts/run-blackbox-tests.sh

```diff
@@ -26,7 +31,27 @@ docker exec montagu_db_1 psql -U vimc -d montagu -c \
        "CREATE USER MAPPING FOR vimc SERVER montagu_db_annex OPTIONS (user 'vimc', password 'changeme');"
 # -------------------------------------------------------------
 
-# Build and image that can run blackbox tests
+
+ORDERLY_IMAGE="vimc/orderly:master"
+OW_MIGRATE_IMAGE="vimc/orderlyweb-migrate:master"
+OW_CLI_IMAGE="vimc/orderly-web-user-cli:master"
+
+# create orderly db
```

Comment:
In TC we could add the demo artifact from the orderly build as a dependency to save having to regenerate it each time (this is what we do in the OW build, and now in the webapps)